### PR TITLE
Fix nits in user-guide-windows-containers

### LINKED ISSUE #28370: Nits in setup/production-environment/windows/user-guide-windows-containers
**This is a Bug Report**

**Problem & Proposed Solution:**

In https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/ ,

```diff
* Create a Kubernetes cluster that includes a 
-[master and a worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
+master and a [worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
```
(Because, according to [here](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-containers-in-kubernetes),
> The Kubernetes control plane, including the master components, continues to run on Linux. There are no plans to have a Windows-only Kubernetes cluster.)

---

In https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/ again,

```diff
By adding a taint to all Windows nodes, nothing will be scheduled on them (that includes existing Linux Pods). 
In order for a Windows Pod to be scheduled on a Windows node, 
-it would need both the nodeSelector to choose Windows, and the appropriate matching toleration.
+it would need both the nodeSelector and the appropriate matching toleration to choose Windows.
```

**Page to Update:**
https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/

/language en

### PR DESCRIPTION
**Pull request by jihoon-seo**
Resolves #28370

### CODE REVIEW

File: content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This guide walks you through the steps to configure and deploy a Windows contain
 ## Before you begin
 
 * Create a Kubernetes cluster that includes a 
-[master and a worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
+master and a [worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes)
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md:29]: Hi @jihoon-seo .
Can you replace the word "master" with "control plane" node?

Comment (jihoon-seo) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md:29]: Sure, I will do that soon!

**Comment by jihoon-seo** [comment]
@kbhawkey Thank you for your comment, and I reflected.
PTAL, and when someone says this PR is good to be merged,
I will do a squash job. 😊

**Comment by perithompson** [comment]
Thanks @jihoon-seo!
/lgtm

**Review by celestehorgan — COMMENTED** [review]
/approve

**Pull request merged**